tests/test_redis_session.py

```
@@ -0,0 +1,251 @@
+        # テスト前にデータクリア
+        try:
+            client.flushdb()
+            yield client
+        except redis.ConnectionError:
+            pytest.skip("Redis test server not available")
+        finally:
+            # テスト後もクリア
+            try:
+                client.flushdb()
+            except:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)